README.md

```diff
@@ -8,34 +8,80 @@
 
   <p align="center">
     <br />
-    <a href="https://codemod.com/automations/">Registry</a>
+    <a href="https://codemod.com/registry/">Registry</a>
     ·
     <a href="https://codemod.com/community">Community</a>
     ·
-    <a href="https://docs.codemod.com">Documentation</a>
+    <a href="https://docs.codemod.com">Docs</a>
   </p>
 </p>
 
 ## Codemod.com
 
-At Codemod.com we are building the ecosystem, platform and tooling around code modification bots, aka codemods. The Codemod platform helps you create, distribute, and deploy codemods in codebases of any size.
+At Codemod.com we are building the ecosystem, platform and tooling around code modification bots, aka codemods. Codemod platform helps you create, distribute, and deploy codemods in codebases of any size.
```

Comment:
No codemod.com. 
Just "At Codemod..."
Or you can rephrase the whole paragraph to use fewer Codemod words. Codemod with capital C is us.